package.json

```diff
@@ -21,7 +21,7 @@
     "url": "https://github.com/plone/volto/issues",
     "email": "plone-developers@lists.sourceforge.net"
   },
-  "homepage": "https://plone.org",
+  "homepage": "http://plone.github.io/volto",
```

Comment:
we will not use this URL. We will use styleguide.voltocms.com but I won't configure it here.

Reply:
ok. I'll remove it. I used it for testing.